src/components/CouponDetails/index.jsx

```diff
@@ -106,7 +110,7 @@ class CouponDetails extends React.Component {
     }, {
       label: 'Remind',
       value: 'remind',
-      disabled: isAssignView || isRedeemedView || selectedCodes.length === 0,
+      disabled: isAssignView || isRedeemedView || (tableData && tableData.count === 0),
```

Comment:
I think it would make sense if all options are disabled when there's no table data. I would recommend creating a new variable:

```js
const hasTableData = tableData && tableData.count;
```
Then, use it in the `disabled` conditional for each of the options.